cardlibrary/ancientstirrings.lua

```diff
@@ -1838,7 +1838,7 @@ local stirrings = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
 	
 	["Lezus 6"] = {
 		["Id"] = 662115933,
-		["Name"] = "Lezus",
+		["Name"] = "Token",
```

Comment:
Why is its name changed to Token?~ Shoudln't its rarity be changed to Token instead?